condor_watch_q.py

```diff
@@ -421,9 +424,10 @@ def table_by_event_log(self, abbreviate_path_components=False):
             d = {js: 0 for js in JobStatus}
             live_job_ids = []
             for cluster_id, proc_statuses in state.items():
+            
                 for proc_status in proc_statuses.values():
-                    d[proc_status] += 1
-
+                    d[proc_status] += 1 #UPDATE THE NUMBER OF EACH STATUS
```

Comment:
I also dislike comments like this. If it isn't clear what this line does from the code, let's change the variable names to make it clear.